NitroxClient/GameLogic/PlayerModelBuilder/HsvColorFilter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.PlayerModelBuilder
+{
+    public class HsvColorFilter
+    {
+        private float targetHue;
+        private float targetSaturation;
+        private float targetVibrance;
+        public List<ColorValueRange> HueValueRanges => new List<ColorValueRange>();
```

Comment:
These getters return a **new** List every time they are evaluated/called - I don't think that's intended?

Reply:
Yeah - dumb last second move. I already fixed this on a different branch and planned to land the fix with that branch.